quarkus/defaults/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+plugins {
+  id("polaris-java")
+  id("polaris-license-report")
```

Comment:
Don't need the license report here, it's this module doesn't bulld a distributable thing.

Reply:
but it will be in Maven as a jar, no?

Reply:
Yea - but the license reports are for distributables, not for every artifact.

Reply:
As in - the Polaris Quarkus Server has one - the Admin tool should have one. But polaris-core for example not.